rllib/examples/multi_agent_independent_learning.py

```diff
@@ -30,10 +33,8 @@ def env_creator(args):
             "num_workers": 2,
             # Method specific
             "multiagent": {
-                "policies": set(env.agents),
-                "policy_mapping_fn": (
-                    lambda agent_id, episode, worker, **kwargs: agent_id
-                ),
+                "policies": env.get_agent_ids(),
```

Comment:
Sorry, same here. Let's translate to `AlgorithmConfig`.

Reply:
done.